### PR TITLE
Code refactor

### PR DESCRIPTION
**Pull request by simonovic86**
- introduce `eslint`
- fix `npm` start

### CODE REVIEW

File: src/peer.js [hunk 1]
```diff
@@ -1,161 +1,179 @@
...
+          Bootstrap: [],
+          Addresses: {
+            Swarm: [
+              '/ip4/0.0.0.0/tcp/4001',
+              '/ip4/0.0.0.0/tcp/4002/ws',
+            ],
+          },
+        },
+      },
+    });
```
Comment (oed) [inline src/peer.js]: Could be changed to `Box.create(provider, opts)`.

File: src/peer.js [hunk 2]
```diff
@@ -1,161 +1,179 @@
...
+
+      const space = await this.box.openSpace(spaceName, {
+        consentCallback: () => true,
+      });
+
+      await space.syncDone;
+
+      const thread = await space.joinThread(threadName, {
+        ghost: true,
+      });
```
Comment (oed) [inline src/peer.js]: Could be changed to `const thread = box.openThread(spaceName, threadName, { ghost: true })`

**Review by oed — COMMENTED** [review]
One suggested change.

File: src/peer.js (hunk 1 above)
Comment (simonovic86) [reply src/peer.js]: makes sense 👍

File: src/peer.js (hunk 2 above)
Comment (simonovic86) [reply src/peer.js]: I'll change it 👍

**Pull request merged**